protocol/discord/types.go

```diff
@@ -149,14 +150,15 @@ type ImportTaskProgress struct {
 type ImportTasks map[ImportTask]*ImportTaskProgress
 
 type ImportProgress struct {
-	CommunityID   string                `json:"communityId,omitempty"`
-	CommunityName string                `json:"communityName"`
-	Tasks         []*ImportTaskProgress `json:"tasks"`
-	Progress      float32               `json:"progress"`
-	ErrorsCount   uint                  `json:"errorsCount"`
-	WarningsCount uint                  `json:"warningsCount"`
-	Stopped       bool                  `json:"stopped"`
-	m             sync.Mutex
+	CommunityImages map[string]images.IdentityImage `json:"communityImages"`
```

Comment:
This is not alphabetically ordered, so it'd make more sense to but the Images after the Name or something (let ID be first)